src/Emogrifier/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -84,11 +91,27 @@ public function render()
      */
     public function renderBodyContent()
     {
-        $bodyNodeHtml = $this->domDocument->saveHTML($this->getBodyElement());
+        $bodyNodeHtml = $this->renderDomDocument($this->getBodyElement());
 
         return \str_replace(['<body>', '</body>'], '', $bodyNodeHtml);
     }
 
+    /**
+     * Renders the DOMDocument as HTML, eliminating any invalid closing tags for void elements.
+     *
+     * @param \DOMNode|null $node Optional parameter to output a subset of the document.
+     *
+     * @return string
+     */
+    protected function renderDomDocument(\DOMNode $node = null)
```

Comment:
We still have the feature switch here (kind of). I propose we have a `removeUnrecognizedClosingTags(string $html): string` (or with a different method name) here. What do you think?

Reply:
This is outdated.  Did you refresh the page?

Reply:
Oh, it indeed is outdated. Strange. I don't know how this happened. Maybe some hiccup with caches and a flaky mobile internet connection. I'll re-review right away.